src/services/codefixes/superFixes.ts

```diff
@@ -0,0 +1,63 @@
+﻿/* @internal */
+namespace ts.codefix {
+    function getOpenBraceEnd(constructor: ConstructorDeclaration, sourceFile: SourceFile) {
+        // First token is the open curly, this is where we want to put the 'super' call.
+        return constructor.body.getFirstToken(sourceFile).getEnd();
+    }
+
+    registerCodeFix({
+        errorCodes: [`TS${Diagnostics.Constructors_for_derived_classes_must_contain_a_super_call.code}`],

```

Comment:
Error codes should not include the `TS` prefix. it should be the same error code we return from getSemantic/getSyntacticDiagnostics calls. if Roslyn asks with a TS, then we should remove this on the VS side of the call.